sample/FOSSASIA15/sessions

```diff
@@ -1,668 +1,650 @@
+        "long_abstract": "Singapore is the host for main offices of many Open Source companies in Asia. RedHat has one of its Asia-Pacific main offices in the city and welcomes the FOSSASIA community before the event starts.",
+        "start_time": "2015-03-12T14:00:00+08:00",
+        "end_time": "2015-03-12T15:00:00+08:00",
+        "session_type": {
+            "id": 1,
+            "name": "Welcome Developers in town at RedHat Office",
+            "length": 1
+        },
+        "track": {
+            "id": 1,
+            "name": "FOSSASIA 2015"
+        },
+        "comments": "",
+        "language": null,
+        "slides": null,
+        "audio": null,
+        "video": null,
+        "microlocation": {
+            "id": 1,
+            "name": "Biopolis, 30 Biopolis Street, Singapore"
+        },
+        "signup_url": null,
+        "speakers": [
+            {
+                "id": null,
+                "name": null,
+                "organisation": "FOSSASIA"
+            }
+        ],
+        "state": "accepted"
+    },
+    {
+        "id": 2,
+        "title": "Visit of Singapore OneMakerGroup at National Design Center",
+        "subtitle": null,
+        "short_abstract": "Short Abstract for the event",
+        "long_abstract": "Let's find out what the developer and maker community is doing in Singapore.",
+        "start_time": "2015-03-12T15:00:00+08:00",
+        "end_time": "2015-03-12T19:00:00+08:00",
+        "session_type": {
+            "id": 2,
+            "name": "Visit of Singapore OneMakerGroup at National Design Center",
+            "length": 4
+        },
+        "track": {
+            "id": 1,
+            "name": "FOSSASIA 2015"
+        },
+        "comments": "",
+        "language": null,
+        "slides": null,
+        "audio": null,
+        "video": null,
+        "microlocation": {
+            "id": 1,
+            "name": "Biopolis, 30 Biopolis Street, Singapore"
+        },
+        "signup_url": null,
+        "speakers": [
+            {
+                "id": null,
+                "name": null,
+                "organisation": "FOSSASIA"
+            }
+        ],
+        "state": "accepted"
+    },
+    {
+        "id": 3,
+        "title": "Pre-Event Meetup: Singapore Steamboat followed by get together at Blujaz Cafe",
+        "subtitle": null,
+        "short_abstract": "Short Abstract for the event",
+        "long_abstract": "Let's experience Singapore Hotpot at Beach Road near Bugis. Afterwards we walk over to a nice bar, when you have arrived in town.",
+        "start_time": "2015-03-12T19:00:00+08:00",
+        "end_time": "2015-03-13T09:00:00+08:00",
+        "session_type": {
+            "id": 3,
+            "name": "Pre-Event Meetup: Singapore Steamboat followed by get together at Blujaz Cafe",
+            "length": 2
+        },
+        "track": {
+            "id": 1,
+            "name": "FOSSASIA 2015"

```

Comment:
@arp95 issue is regarding updating date format in  samples so that it don't crash while testing. So i made required changes to make samples in working state.